src/app.ts

```diff
@@ -95,7 +96,7 @@ const main = async () => {
       startDate
       title
    * There also must be a file with the key `image` which should be the image
-      associated with the flyer.
+      associated with the flyer. The image type must be png.
```

Comment:
Why do you think this is true? I am pretty sure I updated the upload image function to accept any type of file.

Reply:
When I used a JPEG, the file extension gets cut off to .jpe. I haven’t looked at the code in the upload service yet to see why this is the issue but perhaps you might know something about it.